modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/filters/invoker/test/InvokerFilterTest.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.servlet.filters.invoker.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.module.util.SystemBundleUtil;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.util.HashMapDictionaryBuilder;
+import com.liferay.portal.servlet.filters.ignore.IgnoreFilter;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterConfig;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
+
+/**
+ * @author Eric Yan
+ */
+@RunWith(Arquillian.class)
+public class InvokerFilterTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Test
+	public void testOSGIComponentFilterInitParams() {
+		BundleContext bundleContext = SystemBundleUtil.getBundleContext();
```

Comment:
@ericyanLr If it's an integration test, I think you can just use the test module's bundle context. A random example: `ContentDashboardItemFilterProviderRegistryTest`.